src/chrome/content/rules/20min.ch.xml

```diff
@@ -1,30 +1,9 @@
 <!--
 	Other 20min rulesets:
 		- 20min-tv.ch.xml
-		- 20min.ch-mixedcontent.xml
 		- 20minuti.ch.xml
 
-	Nonfunctional hosts in *.20min.ch:
-		- 20min.ch (h)
-		- www.20min.ch (some pages redirect to http;
-			works with "?httpsredirect" parameter, but has mixed content issues then)
-		- deal.20min.ch (m)
-		- m.20min.ch (some pages have mixed content issues)
-		- secretescapes.20min.ch (e)
-		- static01.20min.ch (r)
-		- tilllate.20min.ch (m)
```

Comment:
`secretescapes` and `tilllate` are not covered by the wildcard cert, the comments should be kept.